### PR TITLE
Muncher CLI CMS

### PR DESCRIPTION
**Pull request by Alaev**
This is a small start for CLI CMS that we will use to create, delete, update, find(read)
At the moment you can add user -> admin, user, customer with locaion
Please note that the users are added with fake data (email) password is the same muncher

1. run program with yarn cms add || npm run cms add
screen shot 2017-12-16 at 18 04 17

2. select what role to add
screen shot 2017-12-16 at 18 04 32

### CODE REVIEW

File: cli-cms/constants.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+const fake_customerWithLocation = {
+  body: {
+    customer: {
+      info: {
+        email: faker.internet.email(),
+        password: 'muncher'
+      },
+      location: {
+        name: faker.name.findName(),
+        type: types[faker.random.number({ min: 0, max: 3 })],
```
Comment (andreykishtov) [inline cli-cms/constants.js]: no image?

File: cli-cms/constants.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+const faker = require('faker');
+
+const types = ['restaurant-reservation', 'take-away', 'order-home'];
```
Comment (andreykishtov) [inline cli-cms/constants.js]: why types isnt part of location

File: cli-cms/index.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+const { prompt ,Separator } = require('inquirer');
+const mongoose = require('mongoose');
+const chalk = require('chalk');
+const { createCustomerWithLocation } = require('../server/controllers/createCustomer/');
+const { register } = require('../server/controllers/users/');
+const { fake_customerWithLocation, fake_user } = require('./constants');
+
+
+mongoose.Promise = global.Promise;
+const db = mongoose.connect('mongodb://localhost/muncher', { useMongoClient: true });
```
Comment (andreykishtov) [inline cli-cms/index.js]: maybe create connection as a varible

Comment (Alaev) [reply cli-cms/index.js]: I did, db.close will close the connection

File: cli-cms/constants.js (hunk 1 above)
Comment (Alaev) [reply cli-cms/constants.js]: added imageurl

File: cli-cms/constants.js (hunk 2 above)
Comment (Alaev) [reply cli-cms/constants.js]: it is on the line 13
`type: types[faker.random.number({ min: 0, max: 3 })],`

each location will have one type at the moment

**Pull request merged**